### PR TITLE
Improve point default color

### PR DESCRIPTION
**Pull request by aashish24**

### CODE REVIEW

**Comment by aashish24** [comment]
@jbeezley as discussed, I will let you fix the bugs.

File: src/core/pointFeature.js [hunk 1]
```diff
@@ -242,14 +242,14 @@ geo.pointFeature = function (arg) {
     var defaultStyle = $.extend(
       {},
       {
-        radius: 10.0,
+        radius: 5.0,
         stroke: true,
-        strokeColor: { r: 0.0, g: 1.0, b: 0.0 },
+        strokeColor: { r: 0.851, g: 0.604, b: 0.0 },
         strokeWidth: 2.0,
```
Comment (jeffbaumes) [inline src/core/pointFeature.js]: Can we change this to 1.0?

Comment (aashish24) [inline src/core/pointFeature.js]: I pixel might be too small but I will see how it looks and if it looks reasonable then I am okay with it.

**Comment by jbeezley** [comment]
I fixed the tests.  Here is some comparisons of stroke widths `1`, `1.25`, `1.5`, and `2` with both renderers.  I think `2` is too wide especially for this size of point.  In the past, I have noticed bad antialiasing for `1` pixel wide circles in svg.  I checked various browsers and the antialiasing doesn't look too bad anymore, but I think `1.25` might look a bit better.

http://output.jsbin.com/tikota/13

**Comment by aashish24** [comment]
thansk @jbeezley. If I zoom way in, then you will see that 1.25 and 1.5 both look good. I think  1.5 looks slightly better though but folks may prefer 1.25.

**Comment by aashish24** [comment]
Okay, lets go with 1.25 then.

**Comment by jbeezley** [comment]
Fixed and ready for merge.

**Comment by aashish24** [comment]
:+1:

**Pull request merged**